backend-shared/config/db/util/utils_test.go

```diff
@@ -1,12 +1,321 @@
+				Namespace_name:   "my-namespace",
+				Namespace_uid:    "test-1",
+				EngineCluster_id: gitopsEngineCluster.Gitopsenginecluster_id,
+			}
+			err = dbQueries.CreateGitopsEngineInstance(ctx, &gitopsEngineInstance)
+			Expect(err).To(BeNil())
+
+			// ---------------------------------------------------------------------------
+			// Create Application
+
+			application = db.Application{
+				Name:                    "my-application",
+				Spec_field:              "{}",
+				Engine_instance_inst_id: gitopsEngineInstance.Gitopsengineinstance_id,
+				Managed_environment_id:  managedEnvironment.Managedenvironment_id,
+			}
+			err = dbQueries.CreateApplication(ctx, &application)
+			Expect(err).To(BeNil())
+
+			// ---------------------------------------------------------------------------
+			// Create deploymentToApplicationMapping request data
+
+			deploymentToApplicationMapping = &db.DeploymentToApplicationMapping{
+				Deploymenttoapplicationmapping_uid_id: string(uuid.NewUUID()),
+				Application_id:                        application.Application_id,
+				DeploymentName:                        "my-depl-to-app-mapping",
+				DeploymentNamespace:                   "my-namespace",
+				NamespaceUID:                          "test-1",
+			}
+		})
+
+		AfterEach(func() {
+			// ----------------------------------------------------------------------------
+			By("Delete resources and clean db entries created by test.")
+			// ----------------------------------------------------------------------------
+
+			_, err = dbQueries.DeleteDeploymentToApplicationMappingByDeplId(ctx, deploymentToApplicationMapping.Deploymenttoapplicationmapping_uid_id)
+			Expect(err).To(BeNil())
+
+			_, err = dbQueries.DeleteApplicationById(ctx, application.Application_id)
+			Expect(err).To(BeNil())
+
+			_, err = dbQueries.DeleteGitopsEngineInstanceById(ctx, gitopsEngineInstance.Gitopsengineinstance_id)
+			Expect(err).To(BeNil())
+
+			_, err = dbQueries.DeleteGitopsEngineClusterById(ctx, gitopsEngineCluster.Gitopsenginecluster_id)
+			Expect(err).To(BeNil())
+
+			_, err = dbQueries.DeleteManagedEnvironmentById(ctx, managedEnvironment.Managedenvironment_id)
+			Expect(err).To(BeNil())
+
+			_, err = dbQueries.DeleteClusterCredentialsById(ctx, clusterCredentials.Clustercredentials_cred_id)
+			Expect(err).To(BeNil())
+
+			// Close connection
+			dbQueries.CloseDatabase()
+		})
+
+		It("Should create new DeploymentToApplicationMapping and if called second time, it should return existing resource instead of creating new.", func() {
+
+			// ----------------------------------------------------------------------------
+			By("Create new DeploymentToApplicationMapping resource.")
+			// ----------------------------------------------------------------------------
+			isNewFirst, errFirst := GetOrCreateDeploymentToApplicationMapping(ctx, deploymentToApplicationMapping, dbQueries, log)
+			Expect(errFirst).To(BeNil())
+			Expect(isNewFirst).To(BeTrue())
```

Comment:
```suggestion
			isNew, err := GetOrCreateDeploymentToApplicationMapping(ctx, deploymentToApplicationMapping, dbQueries, log)
			Expect(err).To(BeNil())
			Expect(isNew).To(BeTrue())
```

Reply:
Feel free to ignore these, as it is just added to remove confusion of too many variables!